### DISCUSSION TITLE
Configuring HA Assist and OpenAI to play music on MA speakers

### DISCUSSION THREAD

**Discussion by kanesbit** [music-assistant/support#2047] [Q&A] [answered]
Hello Everyone,

I'm trying to set up OpenAI integration with Music Assistant, so that I can use Assist to say things like "Play Nirvana in the Office"...and have said music play in said location.

I've followed the instructions at this link 
https://music-assistant.io/integration/installation/#openai-features

I've been to JLo's blog and read this link
https://blog.jlpouffier.fr/chatgpt-powered-music-search-engine-on-a-local-voice-assistant/

However, I'm confused about the end bit.  JLo talks about a blueprint on his Blog, and the ability to trigger an automation with Assist, but this doesn't seem to work.

I can see the JSON coming up in my chats with Assist, but it does not play

I configured debug logging in my config.yaml, and below is the message that I get.

![image](https://github.com/music-assistant/hass-music-assistant/assets/6356507/b617f70f-b3f1-4731-804f-23fd581e2380)

I have even tried this in Developer tools and can see that it appears to fire.

![image](https://github.com/music-assistant/hass-music-assistant/assets/6356507/1f7e30a1-ff0a-474c-a838-80c2b89666f8)

What am I missing here?

**Comment by OzGav** [comment]
Don’t follow JLo’s blog as we have built this feature into MA now.pl undo everything you have done and follow this https://music-assistant.io/integration/installation/#openai-features

**Reply by kanesbit** [reply]
Thanks @OzGav - 

I've undone all of the JLo blog changes, deleted the blueprint and associated automations.

I've completely rebooted my HA server.

I can confirm that I have the following file on my local OS
custom_sentences/en/play_media_on_media_player.yaml

Confirming that I am (and already was) getting this output when I type to Home Assistant
![image](https://github.com/music-assistant/hass-music-assistant/assets/6356507/eb8d3d85-ec42-4be8-ab02-38a3fd0bac48)

Where else should I be looking?

**Reply by OzGav** [reply]
Ok so the OpenAI part is working. What problem do you have now then?

**Reply by e10kstarfire** [reply]
Sorry to jump on this but I am getting the same output as above and don't know how to troubleshoot further, I followed https://music-assistant.io/integration/installation/#openai-features

eg:
![image](https://github.com/music-assistant/hass-music-assistant/assets/23463652/2b850407-d075-4ee4-a944-8c4974c6dd74)

Nothing is playing, what logs do I need to examine to see what is happening

**Reply by OzGav** [reply]
HA recently changed something which has broken the play command. This will be fixed soon.

**Reply by e10kstarfire** [reply]
Ah ok thanks, is there a bug or defect I can track please?

**Reply by OzGav** [reply]
Sorry not at the moment we are tracking this one internally as we found it two days ago.

**Reply by OzGav** [reply]
Actually I forgot about this https://github.com/music-assistant/hass-music-assistant/issues/2413

**Reply by OzGav** [reply, marked as answer]
This should all be fixed in the latest MA versions
